examples/set-a-custom-referrer/README.md

```diff
@@ -0,0 +1,27 @@
+# Set a Custom Referrer
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [TestCafe Example Page](https://devexpress.github.io/testcafe/example/)
+
+The [HTTP referrer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) (otherwise known as referer) is a request header that lets a webpage know where the user has navigated from.
+
+This example shows how to set a custom referrer for the specified requests during tests.
+
+Before the test begins a request hook and a request logger are introduced. When requests are made to a webpage, the hook overwrites the requests's 'Referer' header with a desired value.
+
+The test starts on a dummy page and navigates to another page with a `t.navigateTo` method and the request logger captures the requests. The `t.expect.eql` method checks the header value.
```

Comment:
> The test starts ... and navigates ... and the request logger captures ...

I feel like this use of "and" to first join two verbs and then append the second part of the sentence does not sound native to English.

I'd just divide into two sentences:

*The test starts on a dummy page and navigates to another page with a `t.navigateTo` method. The request logger records the requests sent during the test. To verify that the referrer was overwritten, the `t.expect.eql` method checks the header value.*

Reply:
> The test starts on a dummy page and navigates to another page with the `t.navigateTo` method. The request logger records the requests sent during the test. To verify that the referrer was overwritten, the `t.expect.eql` method checks the header value.